src/common/modals/AddInstance/InstanceName.js

```diff
@@ -288,6 +308,67 @@ const InstanceName = ({
           ramAmount ? { javaMemory: ramAmount } : null
         )
       );
+    } else if (isModrinthModpack) {
```

Comment:
Why not just doing `if(modpack?.project_type)` or just `const isModrinthModpack = modpack?.project_type;`

Reply:
I didn't write `if (modpack?.project_type)` because it's less readable. Someone who doesn't know anything about the code will look at that and think "why am I doing this stuff if `project_type` is truthy?".

I also didn't do `const isModrinthModpack = modpack?.project_type;` because the name `isModrinthModpack` implies that it's a boolean, but it could also be a string (because `project_type` is a string). This removes that potential confusion.

I'm a big fan of so-called "self-documenting code", so I try and write in such a way that makes it as easy as possible for other people to read and understand.